src/chunkserver/chunkserver.cpp

```diff
@@ -455,6 +636,13 @@ int ChunkServer::Run(int argc, char** argv) {
         << "Failed to shutdown file pool clean worker.";
     concurrentapply.Stop();
 
+#ifdef WITH_SPDK
+    if (fs::FileSystemType::PFS == lfsOption.type) {
```

Comment:
Why do you need WITH_SPDK, and you need to judge if (fs::FileSystemType::PFS == lfsOption.type)?

Reply:
done